docs/source/conf.py

```diff
@@ -324,6 +324,7 @@
     'os',
     'pathlib',
     'pytorch_wavelets',
```

Comment:
Could you remove pytorch_wavelets and torch from dependencies (and maybe synchronize the deps from setup.py file)? I have rewritten all torch fucntions with cupy and now we dont need these dependencies.